packages/abi-typegen/src/templates/contract/factory.hbs

```diff
@@ -17,4 +17,12 @@ export class {{capitalizedName}}__factory {
   ): {{capitalizedName}} {
     return new Contract(id, _abi, accountOrProvider) as unknown as {{capitalizedName}}
   }
+  static async deployInstance(
```

Comment:
Why `deployInstance` instead of `deployContract` just like the underlying method?

Reply:
Great point, I prefer `deployContract` too. `deployInstance` can actually be confusing as it can mean we are deploying an instance of the factory itself and not a contract _from_ it. 

I think I just went with `deployInstance` because that's the name mentioned in the feedback issue that Nick opened 🙂